src/Game.ts

```diff
@@ -0,0 +1,103 @@
+export class Game {
+    private _lastSymbol: string = ' ';
+    private _board: Board = new Board();
```

Comment:
This Game class is coupled to this specific implementation of a Board. By creating the `Board` class here, we cannot switch out the board for a different implementation.

We could use depenency injection instead, where we pass the Board class to the constructor of Game.